testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -3180,6 +3183,20 @@ protected String createSchemaSql(String schemaName)
         return "CREATE SCHEMA " + schemaName;
     }
 
+    protected TableMetadata getTableMetadata(String catalog, String schema, String tableName)
```

Comment:
revert the move, it's not needed (see other comment)

Reply:
@findepi as discussed above and will wait if this test is not required. and will revert the move.